### DISCUSSION TITLE
GB30 in GBP

### DISCUSSION THREAD

**Discussion by deltamelter** [mampfes/ha_epex_spot#44] [Ideas]
This is an awesome integration and using it for GB 1hour auction prices. The price is GBP but the label is in EUR/MWh.
At 15:45(BST) the half hourly prices are published on the same epex spot page and this is what my energy provider uses.

Would it be possible to have this auction added to the integration? Preferably in with label GBP/MWh, but doesn't matter as it's just the raw numbers I want

**Comment by mampfes** [comment]
Hi, this is an interesting request.

I added basic support for GB-30 in the main branch (no release yet, because some testing is required). Price is still EUR/MWh - I will fix this later. Could you please test it?

But I also have a question: Do you know which time zone is used for the Hours column? Is it BST and GMT?
![image](https://github.com/mampfes/ha_epex_spot/assets/375950/dd439577-fa81-498f-9a22-9de9250e26a8)

**Comment by deltamelter** [comment]
Thanks, I will take a look.

I'm fairly sure the published table (pictured) is in CEST actually.  The local provider uses it in BST so the prices for each day start  23:00-23:30 through to 22:30-23:00 next day which is confusing because it crosses 2 days, but it is manageable.  
At the end of October when DST ends, I assume it will all adjust an reflect the same time range, but I guess we will see :)

**Comment by deltamelter** [comment]
Looking good. The entity labels threw me  a bit because the are not following same pattern but have both GB30 and GB60 configured.
One entity has the same name in both?
![image](https://github.com/mampfes/ha_epex_spot/assets/27061628/c7a055ed-3c91-44ca-b185-4998c0e44f5c)

I'm using both GB30 and GB60 here (wholesale price with the provider fee and peak time hike stacked on top)
![image](https://github.com/mampfes/ha_epex_spot/assets/27061628/300b42e7-ac01-4047-a4bf-d9afd578dc89)

What is the update time? is that adjustable?  The GB60 is usually available within an hour of being published on epex.  I will see when the GB30 updates

**Reply by mampfes** [reply]
I also don't like the new naming conventions... As workaround you can rename the entity_id.

Data will be fetched once per hour (at xx:58).

**Comment by mampfes** [comment]
available in release 2.0.0

**Reply by deltamelter** [reply]
works great.  Thanks
